src/createDijit.ts

```diff
@@ -166,7 +165,7 @@ function resolveCtor<D extends DijitWidget>(Ctor: DijitWidgetConstructor<D> | st
 				handle.remove();
 				reject(error);
 			});
-			const mid = Ctor;
+			const mid = <string> Ctor;
```

Comment:
Lol, so the `next` TS version `v2.1.0-dev.20160815` doesn't narrow the type correctly but the version `v2.1.0-dev.20160817` does.

So I updated this and it worked locally as it used the `20160817` version but for some reason travis used `20160815` version and failed.